src/Maestro/tests/Maestro.ScenarioTests/MaestroScenarioTestBase.cs

```diff
@@ -265,7 +282,32 @@ await WaitForUpdatedPullRequestAsync(targetRepoName, targetBranch)
 
             using (ChangeDirectory(repoDirectory))
             {
-                await ValidatePullRequestDependencies(targetRepoName, pullRequest.HeadBranch, expectedDependencies);
+                await ValidatePullRequestDependencies(targetRepoName, pullRequest.Value.HeadBranch, expectedDependencies);
+
+                if (expectedFeeds != null && notExpectedFeeds != null)
+                {
+                    TestContext.WriteLine("Validating Nuget feeds in PR branch");
+
+                    ISettings settings = Settings.LoadSpecificSettings(@"./", "nuget.config");
+                    PackageSourceProvider packageSourceProvider = new PackageSourceProvider(settings);
+                    IEnumerable<string> sources = packageSourceProvider.LoadPackageSources().Select(p => p.Source);
+
+                    foreach (string feed in expectedFeeds)
```

Comment:
I _think_ this can be `sources.Should().NotContain(notExpectedFeeds)` and `sources.Should().Contain(expectedFeeds)`... I think

Reply:
That would be pretty slick if so, I'll give it a shot!